core/src/main/java/org/infinispan/statetransfer/StateConsumerImpl.java

```diff
@@ -652,7 +651,7 @@ public void start() {
       isFetchEnabled = (mode.isDistributed() || mode.isReplicated()) &&
               (configuration.clustering().stateTransfer().fetchInMemoryState() || configuration.persistence().fetchPersistentState());
       //rpc options does not changes in runtime. we can use always the same instance.
-      rpcOptions = rpcManager.getRpcOptionsBuilder(ResponseMode.SYNCHRONOUS)
+      rpcOptions = rpcManager.getRpcOptionsBuilder(ResponseMode.SYNCHRONOUS_IGNORE_LEAVERS)
```

Comment:
Those options are used to obtain `ClusterListeners`. This operation is safe for sure (since they [fail without any exception](https://github.com/infinispan/infinispan/blob/c6b1c4dbca3eb65f70358d3f2e91b8f550dd5592/core/src/main/java/org/infinispan/statetransfer/StateConsumerImpl.java#L831-L831)). 

However it's also used for [transactions](https://github.com/infinispan/infinispan/blob/c6b1c4dbca3eb65f70358d3f2e91b8f550dd5592/core/src/main/java/org/infinispan/statetransfer/StateConsumerImpl.java#L848-L848). I would still say it's safe since after all we already lost a node...